Tests/Tests.swift

```diff
@@ -1,6 +1,17 @@
 import XCTest
+@testable import SweetSwift
 
 class Tests: XCTestCase {
+    func testDateComponentisation() {
+        // 21st of December 2017 12:00
```

Comment:
What time zone is this in? What will the effect be if the CI server is in a time zone for which this might not be true?

Reply:
That's UTC, I can amend to comment to make it clearer. If the CI server is not running tests on UTC, then I think the fix would be to make sure it does. 👍

Reply:
Adding the comment is fine, but it would probably be helpful to validate that this works independent of UTC.